docs/joss/paper.md

```diff
@@ -69,7 +70,7 @@ quantify the level of turbulent transport encountered in fusion reactors and can
 major drivers of turbulence. The realisation of fusion critically depends on understanding how to
 mitigate turbulent transport, and thus requires high levels of confidence in the predictive tools being
 employed. Many different gyrokinetic modelling codes are available and Pyrokinetics aims to
-standardise the analysis of such simulations.
+standardise the analysis of such computationally demanding simulations.
```

Comment:
Not always true. Pyro works with TGLF and there's no reason you can't run low cost simulations like GA-STD/CBC which don't need to be very expensive, especially when linear

Reply:
Agreed - was trying to emphasize that standardization / accurate inputs/conversions is especially important given runs are particularly challenging!

Reply:
Would it be good to change https://github.com/pyro-kinetics/pyrokinetics/blob/038eb7aee01ce05e5b2fa74334048dc74f2ab57e/README.md?plain=1#L180 to something like Used and Developed by to capture the institutions of all the contributing author?

Reply:
Happy to add institutions but for all the authors we have the citation file